apisupport/maven.apisupport/src/org/netbeans/modules/maven/apisupport/ExamineManifest.java

```diff
@@ -51,7 +51,7 @@ public class ExamineManifest  {
     private String locBundle;
     private boolean publicPackages;
     private boolean populateDependencies = false;
-    private List dependencyTokens = Collections.emptyList();
+    private List<String> dependencyTokens = Collections.<String>emptyList();
```

Comment:
I don't think the generic type passed to the method on the right side is needed. It would be only needed if the compiler can't infer the type in some edge cases. Try without first unless there is a warning (same for the other occurences).

Reply:
Hey @mbien , I thought I had tried doing just that.. But, couldn't remember.. When I do this work, I try to go as fast as possible, so sometimes I do forget.

But, here is the error that I get.. So I guess it's not possible.

```
   [depend] Deleted 2 out of date files in 0 seconds
 [nb-javac] Compiling 2 source files to /home/bwalker/src/netbeans/apisupport/maven.apisupport/build/classes
   [repeat] /home/bwalker/src/netbeans/apisupport/maven.apisupport/src/org/netbeans/modules/maven/apisupport/ExamineManifest.java:54: error: illegal start of type
   [repeat]     private List<String> dependencyTokens = Collections.<>emptyList();
   [repeat]                                                          ^
   [repeat] 1 error
  [nbmerge] Failed to build target: all-maven.apisupport
```

Reply:
no you don't understand. you don't have to specify the generic type at all.

`private List<String> dependencyTokens = Collections.emptyList();`

since you already merged, please clean it up in follow up PRs because specifying generic types for methods is only needed for edge cases, like in lambdas, but not for simple declarations like this.

Reply:
Ugg! Right you are..

I'll work on it..